client/thin-replica-client/include/client/thin-replica-client/grpc_connection.hpp

```diff
@@ -212,6 +225,11 @@ class GrpcConnection {
   std::shared_mutex rss_streams_mutex_;
   std::atomic_uint64_t current_req_id_{1};
 
+  // Reader threads
+  std::unique_ptr<concord::util::ThreadPool> worker_thread_pool_;
```

Comment:
Nit: can we rename `worker_thread_pool_` to `subscription_pool_` for better context?